recipes/phenoshift/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "phenoshift" %}
+{% set version = "0.0.4.dev0" %}
```

Comment:
Please release full versions. This seems ambiquous if it is a dev version or not